backend/src/main/resources/masterdata/templates/en/new-contact-case.txt

```diff
@@ -0,0 +1,49 @@
+Dear Mrs./Mr. {lastName},
+
```

Comment:
German is the default and similar to message.properties the files in root are the default. You can also use a folder "de" but this would be a duplication of the files.

Reply:
No,  I suggest to move the german version also into subfolder. No "Extrawurscht" 8)